dotCMS/src/integration-test/java/com/dotmarketing/portlets/contentlet/business/HostAPITest.java

```diff
@@ -802,10 +810,7 @@ public void Test_Host_With_Multiple_Lang_Versions_Return_Default_Lang_OtherWise_
         final Language mockDefaultLang = mock(Language.class);
         when(mockDefaultLang.getId()).thenReturn(newRandomDefaultLangId);
 
-        final LanguageAPI mockLanguageAPI = mock(LanguageAPI.class);
-        when(mockLanguageAPI.getDefaultLanguage()).thenReturn(mockDefaultLang);
-
-        final HostAPI hostAPIWithMockedLangAPI = new HostAPIImpl(APILocator.getSystemEventsAPI(), mockLanguageAPI);
+        final HostAPI hostAPIWithMockedLangAPI = new HostAPIImpl(APILocator.getSystemEventsAPI()/*, FactoryLocator.getHostFactory()*/);
```

Comment:
why is this `FactoryLocator.getHostFactory()` commented? If not used, please remove it

Reply:
Right, I missed this one